corrector_module/opmon_corrector/database_manager.py

```diff
@@ -232,7 +232,7 @@ def update_document_clean_data(self, document: dict) -> None:
         try:
             db = self.get_query_db()
             clean_data = db[CLEAN_DATA_COLLECTION]
-            clean_data.update({'_id': document['_id']}, document)
+            clean_data.replace_one({'_id': document['_id']}, document)
```

Comment:
Is this an optimization? As I understand the `_id` field is unique so it is fine to assume only one, just curious regarding the change here and below with `remove` -> `delete_one`.

Reply:
This is an API change from `mongopy` library. The `update` method has been deprecated for a while and is removed now.